### ISSUE TITLE
add support for ndt5 and ndt7 runners to self-deployed ndt servers

### ISSUE THREAD

**Issue by critzo** [m-lab/murakami#76]
The current NDT test runners test against the M-Lab platform servers. We also want to support running tests to self-deployed ndt-servers. 

This could be a second set of ndt5/7 runners like `ndt5-self` and `ndt7-self` or additional features to the existing runners to allow specifying a hostname or IP address. 

The desired functionality is that a single Murakami device could test to an M-Lab server, or to one of several self-deployed ndt-servers that are defined in an environment variable or included file that defines the list of servers.

**Comment by laiyi-ohlsen** [comment]
@robertodauria can you add an issue for the longer term solution

**Comment by critzo** [comment]
Updates from testing with local self-deployed server:
* ndt7-client runner supports testing to self-deployed server using the env vars `MURAKAMI_TESTS_NDT7_ENABLED=1` and `MURAKAMI_TESTS_NDT7_HOST=<IP Address:port>
* ndt5-client runner does not respect the same env vars. I rebuilt a murakami image to test and released it to a balena test device, but locate was still used to select the server.

Also, in testing I confirmed that we have not been running any ndt5/7 tests with plaintext, non-TLS. The runners should be configured to **always** run non-TLS or it should be customizable with an env var. Platforms like armv7 and some arm64 devices (like the rpi 4) return TLS measurements significantly lower than non-TLS ones, due to the overhead of running crypto libraries (arm64) or not having them at all (armv7)

These two example command line examples should be used:

* `ndt7-client -format json -quiet -scheme ws -no-verify -server 10.10.20.15:8080`
* `ndt5-client -format=json -quiet -protocol ndt5 -server 10.10.20.15`
